README.md

```diff
@@ -1,32 +1,36 @@
 # Dennis -- helping to release and ship it
 
-## Install
+## Setup
+
+- It's advised you use a cache directory so Dennis doesn't need to clone the repositories for every release.
+
+```
+mkdir -p ~/.dennis
+```
+
+- You may benefit from either adding a Bash alias, or from creating a one-liner script within your project, for running the releases
```

Comment:
Yep, that's done in https://github.com/lystable/lws/pull/1562 and https://github.com/lystable/lystable-frontend/pull/1603

The above docs are more for external users :)